src/PowerShell/Microsoft.WinGet.Client.Engine/Helpers/AppxModuleHelper.cs

```diff
@@ -74,7 +81,11 @@ internal class AppxModuleHelper
         private const string VCLibsUWPDesktopArm = "https://aka.ms/Microsoft.VCLibs.arm.14.00.Desktop.appx";
         private const string VCLibsUWPDesktopArm64 = "https://aka.ms/Microsoft.VCLibs.arm64.14.00.Desktop.appx";
 
+        private const string AppxPackageTemplate = "{0}_{1}_{2}__8wekyb3d8bbwe.appx";
+
         // Xaml
+        private const string XamlPackageName = "Microsoft.UI.Xaml.2.8";

```

Comment:
This is an existing string (`XamlPackage28`), why do we need another name for it?

Reply:
Removed